lemur/authorities/views.py

```diff
@@ -119,6 +120,7 @@ def get(self):
         return service.render(args)
 
     @validate_schema(authority_input_schema, authority_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this is currently controlled via the ADMIN_ONLY_AUTHORITY_CREATION property, so this isn't needed